gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,137 @@
 ---
-description: Proof Market front end guide
+description: The web interface of the Proof Market
 ---
 
-# Front End
+# Web interface
 
 {% hint style="info" %}
 Please note this product is in beta and expected to change.
 {% endhint %}
 
-#### Proof Market front end can be accessed at [**proof.market**](https://proof.market)
+#### The web interface of the Proof Market can be accessed at [**proof.market**](https://proof.market).
 
 ## New user signup
 
-All users who wish to interact with the proof market must create an account.
+All users who wish to interact with the Proof Market must create an account.
```

Comment:
Can be said much simpler.

To interact with the Proof Market, [create an account](link here).
Without an account you can view the exchange, but you need an account to post orders to buy or sell proofs.

And it doesn't look like a callout. It's the main topic of this section, while a callout is something out of the main narrative.

Reply:
fixed